core/validation/towns_fund/schema_validation/schema.py

```diff
@@ -3,12 +3,12 @@
 import logging
 from datetime import datetime
 from enum import EnumType
-from typing import Any
+from typing import Optional
 
 logger = logging.getLogger(__name__)
 
 
-def parse_schema(schema: dict) -> dict[Any, Any]:
+def parse_schema(schema: dict) -> Optional[dict]:
```

Comment:
I can't see anywhere that this can return `None`? Should the return type just be `dict`?

Reply:
Changed it back to dict[Any, Any]